env_file_app_travis

```diff
@@ -28,6 +38,7 @@ ONYPHE_KEY=test
 GREYNOISE_API_KEY=test
 INTELX_API_KEY=test
 WHOISXMLAPI_KEY=test
+UNPAC_ME_API_KEY=test
```

Comment:
There was a typo in this name and that's why the travis build was failing.

Reply:
I was never, ever, going to find this. Thank you.